### PR TITLE
Prevent property or room/house edits from being lost to a tab change

### PR DESCRIPTION
**Pull request by salvoscala**
When a user is 
a. editing a location/property (/admin/bat/config/property/manage/%)
OR
b. editing a room/house type (/admin/bat/config/types/manage/%/edit)

and the user clicks on another tab:

Add a pop-up warning - “Your changes will be lost if you don’t Save” - two options:
'Continue without saving' - takes them to the tab they clicked on
'Cancel' - keep the user on the same tab

### CODE REVIEW

File: modules/roomify/roomify_offers/roomify_offers.module [hunk 1]
```diff
@@ -418,7 +418,6 @@ function roomify_offers_manage_offers_calendar() {
   $links = array();
 
   $css_files = array(
-    drupal_get_path('module', 'roomify_offers') . '/css/offers_pricing.css',
```
Comment (acrollet) [inline modules/roomify/roomify_offers/roomify_offers.module:421]: @salvoscala should offers_pricing.css be removed?

**Review by acrollet — APPROVED** [review]
code looks good, just the one question

File: modules/roomify/roomify_offers/roomify_offers.module (hunk 1 above)
Comment (salvoscala) [reply modules/roomify/roomify_offers/roomify_offers.module:421]: yup, that file doesn't exist :)

Comment (acrollet) [reply modules/roomify/roomify_offers/roomify_offers.module:421]: 🤦‍♂️ ok let's merge

**Pull request merged**